docs/presentation/Content/ContentModifier.md

```diff
@@ -1,32 +1,87 @@
-| `calculatedProperties` | No        | `CalculatedPropertiesSpecification[]`                                | `[]`    | Specifications of calculated properties whose values are generated using provided ECExpressions. *See [this page](./CalculatedPropertiesSpecification.md) for more details*                                                                                |
-| `propertyCategories`   | No        | `PropertyCategorySpecification[]`                                    | `[]`    | Specifications for custom categories. Simply defining the categories does nothing - they have to be referenced from `PropertySpecification` defined in `propertyOverrides` by `id`. *See [this page](./PropertyCategorySpecification.md) for more details* |
-| `propertyOverrides`    | No        | `PropertySpecification[]`                                            | `[]`    | Specifications for various property overrides. *See [this page](./PropertySpecification.md) for more details*                                                                                                                                              |
-
-## Example
-
-```JSON
-{
-  "ruleType": "ContentModifier",
-  "class": { "schemaName": "BisCore", "className": "Element" },
-  "requiredSchemas": [{ "name": "BisCore", "minVersion": "1.0.1" }],
-  "propertyOverrides": [{
-    "name": "Model",
-    "isDisplayed": false,
-  }],
-}
+| Name                                                      | Required? | Type                                  | Default |
+| --------------------------------------------------------- | --------- | ------------------------------------- | ------- |
+| *Picking attributes*                                      |
+| [`requiredSchemas`](#attribute-requiredschemas)           | No        | `RequiredSchemaSpecification[]`       | `[]`    |
+| [`class`](#attribute-class)                               | No        | `SingleSchemaClassSpecification`      | ``      |
+| [`priority`](#attribute-priority)                         | No        | `number`                              | `1000`  |
+| *Content Modifiers*                                       |
+| [`relatedProperties`](#attribute-relatedproperties)       | No        | `RelatedPropertiesSpecification[]`    | `[]`    |
+| [`calculatedProperties`](#attribute-calculatedproperties) | No        | `CalculatedPropertiesSpecification[]` | `[]`    |
+| [`propertyCategories`](#attribute-propertycategories)     | No        | `PropertyCategorySpecification[]`     | `[]`    |
+| [`propertyOverrides`](#attribute-propertyoverrides)       | No        | `PropertySpecification[]`             | `[]`    |
```

Comment:
I suppose it would be good if they linked to something, but I'm leaning towards markdown page for the type rather than TS type. The markdown has a link to TS type.

Reply:
I suppose it would be good if they linked to something, but I'm leaning towards markdown page for the type rather than TS type. The markdown has a link to TS type.